src/assets/stylesheets/loader.scss

```diff
@@ -89,3 +89,13 @@
     top: -0.75em;
   }
 }
+.loadedText {
+    margin: 1em 0 0 0;
+    color: green;
+    bottom: -10px;
+}
+.loadingText {
+    margin: 1em 0 0 0;
+    color: gray;
+    bottom: -10px;
```

Comment:
Are these `bottom` rules even doing anything? I don't think that means anything outside of the context of elements with a specified `position`.

Reply:
You should have three CSS classes, one called e.g. `status` which means "the thing positioned underneath the logo", which has the margin applied, and two more called `loaded` and `loading` which control the color. (Or, alternatively, have gray be the default color and a class called `finished` which applies the green color.) Then the element should always have the `status` class and it should choose which of the `loading`/`loaded`/`finished` classes it does or doesn't have at runtime based on what's going on.